guides/v2.3/rest/bulk-endpoints.md

```diff
@@ -9,6 +9,12 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
+{: .bs-callout .bs-callout-warning}
+By default, Magento uses RabbitMQ (AMQP) for processing message queues. Before using Bulk operations, you should see [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
+
+{:.bs-callout .bs-callout-info}
+Alternatively you can use CRON for processing message queues without RabbitMQ installed, see [Manage message queues]({{ page.baseurl }}/config-guide/mq/manage-message-queues.html).
```

Comment:
So I would leave it

Reply:
@hostep  and @nuzil   Looking at an issue opened in Magento code repo: https://github.com/magento/magento2/issues/25142#issuecomment-544003657

In that issue, it looks like customer gets an error if they don't use RabbitMQ with Bulk API.   Adding the note about using alternatives for processing message seems to add confusion. Even if it is possible to use something besides RabbitMQ to process messages with the Bulk API, isn't the advantage of the Bulk API is that it is pre-configured with the functionality that you need? 

My suggestion is to delete the "Alternatively, you can use ... " note from this topic for now.  And to provide information about the custom configuration that does not use RabbitMQ as a separate topic, perhaps a KB article or blog.  That is assuming that the best practice and easiest way to process message queues using the Bulk API is to use the default configuration.

Additionally, after we close this PR, it might be helpful to update the [Bulk API tutorial](https://devdocs.magento.com/guides/v2.3/rest/tutorials/bulk-configurable-product/config-product-intro.html) to include RabbitMQ configuration in the _Before you begin_ section.